src/screens/HomeScreen.js

```diff
@@ -74,6 +74,8 @@ export const HomeScreen = ({ navigation, route }) => {
     undefined,
     globalSettings?.settings?.pushNotifications
   );
+
+  useTrackAppStart();
```

Comment:
is this the first possible spot to run this?
when entering the home screen, we track that separately already.
the app start should be earlier in my understanding.

Reply:
i changed it to be called immediately upon instance creation.

the hook is then unused so i removed it

Reply:
thanks, makes more sense to me. i changed `info` to `userInfo` per rebase. so update you local branch first in terms of updating something else.

Reply:
✅